### ISSUE TITLE
Display free-text content for collDate

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cdc.versions#243]
**Original report on BitBucket by Taina Jääskeläinen.**

----------------------------------------

Extract and display in detailed study description free-text content from collDate even if there is @&zwnj;date attribute.

This is because in cross-national combined data files, the @&zwnj;dates portray the first collection date in any country and the last collection date in any country. The country-specific information on collection dates may be entered into free-text content.

The free-text are only displayed, and the filter relies on @&zwnj;dates as before.

In short, display free-text content of:

* <stdyDscr><stdyInfo><sumDscr>\[content displayed\]<collDate event="single">@&zwnj;xml:lang
* <stdyDscr><stdyInfo><sumDscr>\[content displayed\]<collDate event="start">@&zwnj;xml:lang
* <stdyDscr><stdyInfo><sumDscr>\[content displayed\]<collDate event="end">@&zwnj;xml:lang

Even if there is information in the @&zwnj;date.

Hopefully this can be done by the Jan 2021 release.
